pkgs/development/libraries/onnxruntime/default.nix

```diff
@@ -19,10 +19,17 @@
 , gtest
 , protobuf3_21
 , pythonSupport ? true
+, cudaSupport
```

Comment:
Grepping the repo I see instances of both, with about half-half split. The docs https://github.com/NixOS/nixpkgs/blob/0bccbcf89c54df9910e672249955926dd1f16209/doc/languages-frameworks/cuda.section.md?plain=1#L15-L19 recommend `cudaSupport ? config.cudaSupport`, so that's what I used.